src/NServiceBus.AcceptanceTesting/Support/ScenarioRunner.cs

```diff
@@ -210,7 +210,7 @@ static void PrintSettings(IEnumerable<KeyValuePair<string, object>> settings)
                             }
                         }
 
-                        await Task.Delay(1).ConfigureAwait(false);
+                        await Task.Yield();
```

Comment:
So, the actual practical difference here is that `Yield` will always be an async continuation, but `Delay` might not be, and likely isn't since the delay amount is so small?

In that case, would a larger delay value have a similar effect?